codegen/smithy-ruby-codegen-test/model/component-test/compression.smithy

```diff
@@ -0,0 +1,35 @@
+$version: "1.0"
+namespace smithy.ruby.tests
+
+@requestCompression(
+    encodings: ["gzip"]
+)
+
+@http(method: "POST", uri: "/some_operation")
+operation SomeOperation {
+    input: SomeOperationInput
+}
+
+@http(method: "POST", uri: "/some_streaming_operation")
+operation SomeStreamingOperation {
+    input: SomeStreamingInput
+}
+
+@input
+structure SomeOperationInput {
+    @httpPayload
```

Comment:
Nice - this is the trait we will want to look for in the fakeProtocol builderGenerator and then put this member on the request body.